cli/src/command.rs

```diff
@@ -293,6 +304,31 @@ where
 pub fn run() -> Result<()> {
 	let cli: Cli = Cli::from_args();
 
+	#[cfg(feature = "pyroscope")]
+	let mut pyroscope_agent_maybe = if let Some(ref agent_addr) = cli.run.pyroscope_server {
+		let address = agent_addr
```

Comment:
Do we need an address resolution here in the first place, counting that we reconstruct an URL afterwards? This URL can support more than plain `http`, for example `https`. I have no strong opinion here, but maybe it is better to name option like `pyroscope_url` and treat it as an URL from the beginning?

Reply:
Two competing things: We want compatibility, but we also want URLs. `url::Url` does not accept an IP without a prefixed protocol such as `tcp://` or `http://` - but we must maintain compatibility. The above is an escape hatch to reduce either to an IP address and avoid the ordeal.